components/input.css

```diff
@@ -0,0 +1,26 @@
+@import url(/stylesheets/main.css);
+
+.input {
+  display: block;
+  padding: 0.5rem;
+  width: 15rem;
+  margin-top: 5px;
+}
```

Comment:
This makes all inputs as block level. In case you want to have two inputs side by side you may have to wrap in a container and flex them

Reply:
I did not think of it from that point of view. will make necessary changes.

Reply:
the confusion was created because I accidentally merged the branch to dev. will look into the files

Reply:
addressed the changes in latest commits